platformio.ini

```diff
@@ -15,13 +15,8 @@ framework = arduino
 monitor_speed = 115200
 test_port = COM3
 lib_deps = 
-    fastled/FastLED @ ^3.4.0
-    https://github.com/bigsdawg-2k/command_buffer.git
-
-; Debug tool = ESP-PROG
+	https://github.com/bigsdawg-2k/command_buffer.git
+	fastled/FastLED @ ^3.5.0
```

Comment:
For whatever reason FastLED 3.4.0 doesn't compile with the esp32 Arduino headers that platformIO pulled down.  I needed to update to 3.5.0